### ISSUE TITLE
can't install

### ISSUE THREAD

**Issue by conor-o-brien** [lukiffer/haiku#18]
Hi when trying install on hassbian I get the following error, any ideas what to do, thanks.

`npm install @haiku-ui/haiku
npm ERR! code ENOSELF
npm ERR! Refusing to install package with name "@haiku-ui/haiku" under a package
npm ERR! also called "@haiku-ui/haiku". Did you name your project the same
npm ERR! as the dependency you're installing?
npm ERR!
npm ERR! For more information, see:
npm ERR!     <https://docs.npmjs.com/cli/install#limitations-of-npms-install-algorithm>

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/homeassistant/.npm/_logs/2018-10-07T18_09_25_900Z-debug.log
(homeassistant) `

**Comment by lukiffer** [comment]
**TL;DR** I no longer think NPM is the appropriate deployment medium for this, so I'm going to focus more on making it easier to deploy than fixing this specific issue.

Firstly, apologies for the super-delayed response. Work has been crazy for the last quarter and am just now starting to be able to focus on this again. Which is a bit awkward as I just upgraded HA to 0.86.1 and everything is busted... it looks like they started URL encoding import statements and the unpkg host doesn't like it. I digress.

But now that more of Lovelace has been fleshed out and (at least somewhat) finalized, I don't think I'll be using NPM as a deployment medium. It was really a shortcut on my part to try and avoid hosting the compiled files.

I'm considering moving from Sass to CSS so I can simply reference the CSS from each web component and use githubusercontent to host raw files, or still compile them and just throw them in an S3 bucket. This would mean that no files would need to be installed in your HA instance, and the YAML configuration would just be something like:

```yaml
resources:
  - url: https://some-s3-bucket.s3.amazonaws.com/haiku/cards/room-card.js
    type: module
```
